<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Connection</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="text_port">
           <property name="text">
            <string>5678</string>
           </property>
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
         <item>
          <widget class="QPushButton" name="button_start">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_stop">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>GPIOs</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_status">
         <property name="text">
          <string>disconnected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_gpios">
         <property name="text">
          <string>0x00</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_stop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_about()</slot>
  <slot>on_start()</slot>
  <slot>on_stop()</slot>
 </slots>
</ui>
